### DISCUSSION TITLE
My Github Student was not Activate after 3 days

### DISCUSSION THREAD

**Discussion by Gunstacker** [community/community#137833] [GitHub Education]
### Hi everyone,

Institution: Pontifical Catholic University of Goiás

It was recommended that I open a question if it was not activated after 3 days, I have already sent all the documents from my academic institution and it has not yet been activated after 3 days, I hope that with this post you will soon enable my right as a student and provide me with the benefit that my institution offers me.

**Comment by davevad93** [comment]
Hi @Gunstacker ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Gunstacker was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
